code/modules/projectiles/guns/throw/crossbow.dm

```diff
@@ -29,41 +21,48 @@
 	if(cell && severity)
 		emp_act(severity)
 
-/obj/item/gun/throw/crossbow/update_icon()
-	if(!tension)
-		if(!to_launch)
-			icon_state = "[initial(icon_state)]"
-		else
-			icon_state = "[initial(icon_state)]-nocked"
-	else
-		icon_state = "[initial(icon_state)]-drawn"
+obj/item/gun/throw/crossbow/update_icon()
```

Comment:
I think you need to call `..()` somewhere in this

Also, you need a `/` in front of `obj`

Reply:
done , thanks for helping me